spec/measure_repository_service_test_kit/read_by_id_spec.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+RSpec.describe MeasureRepositoryServiceTestKit::ReadById do
+  let(:suite) { Inferno::Repositories::TestSuites.new.find('measure_repository_service_test_suite') }
+  let(:group) { suite.groups[1] }
+  let(:session_data_repo) { Inferno::Repositories::SessionData.new }
+  let(:test_session) { repo_create(:test_session, test_suite_id: suite.id) }
+  let(:url) { 'http://example.com/fhir' }
+  let(:error_outcome) { FHIR::OperationOutcome.new(issue: [{ severity: 'error' }]) }
+
+  def run(runnable, inputs = {})
```

Comment:
I think we'll wind up needing to run this `run` function in every rspec file. Is it worth moving this to a `util` directory under `rspec`?